src/containers/Auth/Login/Login.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+import * as actions from '../../../store/actions/index'
+
+import '../../../assets/semantic/semantic.min.css';
+import { Form, Input, Button } from 'semantic-ui-react';
+import Aux from '../../../hoc/aux';
+
+
+
+class Login extends Component {
+    state = {
+        email: '',
+        password: ''
+    }
```

Comment:
No error messages displayed on the wrong user trying to login
![Desktop screensh](https://user-images.githubusercontent.com/39910089/110081358-d3247f80-7db1-11eb-93ff-0a679fab5e6b.png)